bin/sbank_pipe

```diff
@@ -36,14 +37,6 @@ from glue.pipeline import DeepCopyableConfigParser as dcConfigParser
 from glue import pipeline
 from lalapps import inspiral
 
-def which(prog):
-    which = subprocess.Popen(['/usr/bin/which', prog], stdout=subprocess.PIPE)
-    out = which.stdout.read().strip()
-    if not out:
```

Comment:
Note that the error check is gone now. Is the downstream code going to handle a missing executable in a sane way (i.e. not create a workflow of 1000 jobs which will all immediately fail)?

Reply:
(`shutil.which()` returns None for missing executables, BTW)

Reply:
*sigh*, fair point as well. I've reintroduced the function with error checking!